common/filestore_utils.py

```diff
@@ -26,7 +29,23 @@ def _using_gsutil():
     except KeyError:
         return True
 
-    return experiment_filestore_path.startswith('gs://')
+    return is_filestore_path_gcs(experiment_filestore_path)
+
+
+def is_filestore_path_gcs(filestore_path):
+    """Returns True if |filestore_path is a GCS URL. Assumes that GCS paths are
+    using gs:// and not http."""
+    return filestore_path.startswith(GCS_GSUTIL_PREFIX)
+
+
+def get_user_facing_path(filestore_path):
```

Comment:
name is slightly confusing, get_web_url ?

Reply:
get_web_url is misleading because it returns a unix path if filestore_path is a unix path.